packages/cli/package.json

```diff
@@ -24,8 +24,10 @@
     "dist",
     "src"
   ],
+  "peerDependencies": {
+    "@compiled/react": ">= 0.4.7"
```

Comment:
ohhh - was it using its own dependency before?

Reply:
Not exactly. It was not used. I just added it. It was anyways taking from cwd and that will be the directory where we are running this cli. So made it peer so that it ll warn if user tries to install cli in the repo. Otherwise if they are using `npx`, they can go through the Readme file for the instructions.